conf.yaml

```diff
@@ -122,7 +122,7 @@ contents:
             prefix:     en/welcome-to-elastic
             current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ 8.1 ]
+            branches:   [ 8.2 ]
```

Comment:
I'm not sure if we still want to publish an 8.1 version of this book, too. 

cc: @DanRoscigno @nrichers

Reply:
Intuitively, it feels like this book should always be at the latest stack version similar to how our SaaS docs only have one version available (the latest one). Also, the 8.1 version is arguably far less desirable than the 8.2 one, so I'd be fine with removing.

Reply:
Sounds good to me. Thanks @nrichers .